tests/Odbo/OdboTestCase.php

```diff
@@ -38,4 +38,19 @@ protected function assertTimestampsEqualsWithDelta(DateTime|string $expected, Da
 
         $this->assertEqualsWithDelta($expected->getTimestamp(), $actual->getTimestamp(), $delta);
     }
+
+    /**
+     * @test
+     * @runInSeparateProcess
+     * @covers \buildDefaultPdoObject
+     */
+    public function testPdoAttributesArray(): void
+    {
+        define('__OBRAY_DATABASE_ATTRIBUTES__', [
+            PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION,
+        ]);
```

Comment:
You might consider an attribute different than the default one we provide to validate this more effectively.

Reply:
[abc3df3](https://github.com/MaloufSleep/obray/pull/28/commits/abc3df3ca16aab0e86009cddc6967a168df4bec7)